clj-refactor.el

```diff
@@ -761,14 +761,17 @@ A new record is created to define this constructor."
 ;; ------ file -----------
 
 (defun cljr--project-dir ()
-  (or (ignore-errors
-        (file-truename
-         (locate-dominating-file default-directory "project.clj")))
-      (ignore-errors
-        (file-truename
-         (locate-dominating-file default-directory "build.boot")))
-      (ignore-errors (file-truename
-                      (locate-dominating-file default-directory "pom.xml")))))
+  (let ((project-dir (or (ignore-errors
+			   (file-truename
+			    (locate-dominating-file default-directory "project.clj")))
+			 (ignore-errors
+			   (file-truename
+			    (locate-dominating-file default-directory "build.boot")))
+			 (ignore-errors (file-truename
+					 (locate-dominating-file default-directory "pom.xml"))))))
+    (if (null project-dir)
```

Comment:
Good point.

Changed it as per your comment.

I was more on the 'drive' to complete the whole task, so I've missed the obvious stuff like this. Thanks